CHANGELOG.md

```diff
@@ -1,3 +1,62 @@
+## Release 0.12.0
+
+New features and enhancements:
+- Details panel redesign
+  [#752](https://github.com/weaveworks/scope/pull/752)
+- k8s: Use service account token by default and improve error logging
+  [#808](https://github.com/weaveworks/scope/pull/808)
+- Probes now use TLS against scope.weave.works by default
+  [#785](https://github.com/weaveworks/scope/pull/785)
+- Gather per-process CPU and memory metrics.
+  [#767](https://github.com/weaveworks/scope/pull/767)
```

Comment:
We usually sort the list by most impressive (for users).  So I'd put this #2, and probably put all the k8s stuff together,